ci/meta.yml

```diff
@@ -320,3 +320,80 @@ meta:
               do
                 ./fly --target concourse land-worker --worker $worker;
               done
+
+    create-users:
+      task: create-users
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((docker-awscli.repository))
+            version: ((docker-awscli.version))
+            tag: ((docker-awscli.version))
+        params:
+          AWS_PROFILE: ci
+          AWS_SHARED_CREDENTIALS_FILE: .aws/credentials
+          AWS_CONFIG_FILE: .aws/config
```

Comment:
Why are we going back to this "old" pattern, and not following the `source /assume-role` pattern as per https://git.ucd.gpn.gov.uk/dip/aws-common-infrastructure/wiki/AWS-Concourse-Pipeline-Migration-Guide#on-the-subject-of-credentials? (admittedly, that doc could do with a fully-fleshed example!).

Reply:
this is running on Crown Concourse

Reply:
> this is running on Crown Concourse

:this: